### ISSUE TITLE
Exception when crafting with a player corpse nearby

### ISSUE THREAD

**Issue by anderskronquist** [abearcodes/Valheim#3]
I receive the following exception when I attempt to use a crafting table with a player corpse nearby:

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
ABearCodes.CraftingWithContainers.Tracking.TrackContainersWhenPlayerUsesCraftingStationPatch+<>c__DisplayClass5_0.<FindAllowedContainersInCraftingStationRange>b__1 (<>f__AnonymousType0`2[<Container>j__TPar,<OwningPiece>j__TPar] context) (at <d0d0ce5b9c0f4349b4e657dfa21a38c6>:0)
System.Linq.Enumerable+WhereSelectEnumerableIterator`2[TSource,TResult].ToList () (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
ABearCodes.CraftingWithContainers.Tracking.TrackContainersWhenPlayerUsesCraftingStationPatch.Prefix (Player& __instance, CraftingStation station) (at <d0d0ce5b9c0f4349b4e657dfa21a38c6>:0)
(wrapper dynamic-method) Player.DMD<Player::SetCraftingStation>(Player,CraftingStation)
CraftingStation.Interact (Humanoid user, System.Boolean repeat) (at <0d0bfdea58e44fd98ffc2cf4207dbf16>:0)
Player.Interact (UnityEngine.GameObject go, System.Boolean hold) (at <0d0bfdea58e44fd98ffc2cf4207dbf16>:0)
Player.Update () (at <0d0bfdea58e44fd98ffc2cf4207dbf16>:0)

```

**Comment by abearcodes** [comment]
Fixed in 1.0.3. Will release soon and will close the ticket.

**Comment by abearcodes** [comment]
1.0.3 released
